internal/background/availability_queue.go

```diff
@@ -60,6 +60,8 @@ func sendAvailabilityRequestMessages(ctx context.Context, batchSize int, tickDur
 				send(ctx, messageBuffer...)
 				messageBuffer = messageBuffer[:0]
```

Comment:
Ah, right .. no point of cleaning the buffer when you return right after :) well that's what linter says

Reply:
@lzap can you pls remove this line so we can merge this? :)

Reply:
Hmm weird check, but okay. I can live with that although my guts tell me the buffer should really be cleared... :-)